lib/DDG/Spice/SimilarSites.pm

```diff
@@ -15,17 +16,20 @@ topics "everyday", "special_interest";
 attribution github => ['https://github.com/Adman', 'Adman'],
             twitter => ['http://twitter.com/adman_X', 'adman_X'];
 
-triggers startend => "similar sites", "similar sites to", "similar web to", "sites like", "websites like", "webs like", "pages like";
+triggers start => ["sites like", "site like", "websites like", "website like", "webs like", "pages like"];
+
+triggers any => ["similar sites", "similar web", "websites similar", "website similar", "similar websites", "similar website"];
 
 spice to => 'http://www.similarsitesearch.com/api/similar/$1';
 
 spice wrap_jsonp_callback => 1;
 
-handle remainder => sub {
-    s/^https?:\/\///;
-
+handle remainder_lc => sub {
+    $_ =~ s/(to|like|\s)*//g;
```

Comment:
This doesn't look reliable. Consider the query "similar sites to like.com". Please add this to the test as well.